{{cookiecutter.repo_name}}/{{cookiecutter.webapp_name}}/urls.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2017 University of Dundee.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from django.conf.urls import url
```

Comment:
This import fails now on Django 4.2. It needs to be `from django.urls import re_path` and then replace `url()` with `re_path()` below.